css/properties/-moz-user-focus.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "-moz-user-focus": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/CSS/-moz-user-focus",
```

Comment:
404, probably should be https://developer.mozilla.org/docs/Web/CSS/-moz-user-focus?